siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -0,0 +1,1644 @@
+            'RxEnbl-SP.B2', 'RxEnbl-RB.B2',
+            'RxEnbl-SP.B3', 'RxEnbl-RB.B3',
+            'RxEnbl-SP.B4', 'RxEnbl-RB.B4',
+            'RxEnbl-SP.B5', 'RxEnbl-RB.B5',
+            'RxEnbl-SP.B6', 'RxEnbl-RB.B6',
+            'RxEnbl-SP.B7', 'RxEnbl-RB.B7',
+            'RxLockedLtc-Mon', 'RxLockedLtcRst-Cmd',
+            ])
+        # interlock callback
+        pvo = self._evg_dev.pv_object('IntlkEvtStatus-Mon')
+        pvo.auto_monitor = True
+        pvo.add_callback(self._callback_evg_intlk)
+        pvo.connection_callbacks.append(self._conn_callback_timing)
+        # rxlock callback
+        pvo = self._evg_dev.pv_object('RxLockedLtc-Mon')
+        pvo.auto_monitor = True
+        pvo.add_callback(self._callback_evg_rxlock)
+
+        # # Fouts
+        foutnames = list(self._const.FOUTS_2_MON) + \
+            list(self._const.FOUTSFIXED_RXENBL.keys())
+        self._thread_cbfout = {fout: None for fout in foutnames}
+        self._fout_devs = {
+            devname: _Device(
+                devname,
+                props2init=[
+                    'RxEnbl-SP', 'RxEnbl-RB',
+                    'RxLockedLtc-Mon', 'RxLockedLtcRst-Cmd',
+                ], auto_monitor_mon=True)
+            for devname in foutnames}
+        self._fout2rxenbl = dict()
+        for devname, dev in self._fout_devs.items():
+            pvo = dev.pv_object('RxLockedLtc-Mon')
+            pvo.add_callback(self._callback_fout_rxlock)
+            pvo.connection_callbacks.append(self._conn_callback_timing)
+            rxenbl = self._const.FOUTSFIXED_RXENBL.get(devname, 0)
+            self._fout2rxenbl[devname] = rxenbl
+
+        # # AFC timing
+        self._afcti_devs = {
+            idx+1: _Device(
+                f'IA-{idx+1:02}RaBPM:TI-AMCFPGAEVR',
+                props2init=[
+                    'DevEnbl-Sel', 'DevEnbl-Sts',
+                    'RTMClkLockedLtc-Mon', 'ClkLockedLtcRst-Cmd',
+                    'RTMClkRst-Cmd',
+                    'RTMPhasePropGain-SP', 'RTMPhasePropGain-RB',
+                    'RTMPhaseIntgGain-SP', 'RTMPhaseIntgGain-RB',
+                    'RTMFreqPropGain-SP', 'RTMFreqPropGain-RB',
+                    'RTMFreqIntgGain-SP', 'RTMFreqIntgGain-RB',
+                ], auto_monitor_mon=True)
+            for idx in range(20)
+        }
+        for dev in self._afcti_devs.values():
+            pvo = dev.pv_object('RTMClkLockedLtc-Mon')
+            pvo.connection_callbacks.append(self._conn_callback_timing)
+
+        # # RF EVE
+        trgsrc = _HLTimeSearch.get_ll_trigger_names('SI-Glob:TI-LLRF-PsMtm')
+        pvname = _LLTimeSearch.get_channel_output_port_pvname(trgsrc[0])
+        self._llrf_evtcnt_pvname = f'{pvname}EvtCnt-Mon'
+        self._everf_dev = _Device(
+            pvname.device_name,
+            props2init=[self._llrf_evtcnt_pvname, ],
+            auto_monitor_mon=True)
+        pvo = self._everf_dev.pv_object(self._llrf_evtcnt_pvname)
```

Comment:
I believe this is wrong?

```
In [17]: trgsrc = HLTimeSearch.get_ll_trigger_names('SI-Glob:TI-LLRF-PsMtm')

In [18]: pvname = LLTimeSearch.get_channel_output_port_pvname(trgsrc[0])

In [19]: evtcnt = f'{pvname}EvtCnt-Mon'

In [20]: dev = Device(pvname.device_name, props2init=[evtcnt])

In [21]: dev.pvnames
Out[21]: {'RA-RaSIA01:TI-EVE:RA-RaSIA01:TI-EVE:OTP01EvtCnt-Mon'}
```

The correct initialization for `self._everf_dev` would be `dev = Device(pvname.device_name, props2init=['OTP01EvtCnt-Mon'])`

And then the  `dev.pv_object` call should use just `OTP01EvtCnt-Mon` as well, not the whole PV name.

Reply:
thanks, fixed!